src/config/2015.php

```diff
@@ -4,18 +4,234 @@
+            'presenters' => ['josiah_baker'],
+            'description' => ' A pretty in depth description of how historic ad activity can be used to create a regression model which would then be able to answer questions like "How do photos increase my clicks/leads compared to ads without photos," or "What is the impact of making my ad a featured or premium ad."  Details on how the model is built and then how it works to predict clicks and leads on new ads.',
+        ],
+        'session1' => [
+            'title' => 'Visualizing Data',
+            'presenters' => ['alena_devan'],
+            'description' => ' The growing amount of data being collected and analyzed requires innovative data platform solutions. In this talk I will review some of the available technologies to store, manipulate, and visualize data.',
+        ],
+        'what_is_data_science' => [
+            'title' => 'What is Data Science',
+            'presenters' => ['josiah_baker'],
+            'description' => ' A high level discussion of what Data Science is and how it applies to the Web Advertising industry.  Would include brief history, notable examples in other industries, and examples of how it is being used today here at Dominion Enterprises.',
+        ],
+        'making_bad_ass_users_' => [
+            'title' => 'Making Bad@$$ Users.',
+            'presenters' => ['kelley_howell'],
+            'description' => ' This talk will show IT and product teams how to design their applications for mastery. Everyone talk about designing for emotion or designing for delight. But those are concepts that are either too broad or too superficial. What we really want to do is design for mastery - or, turn our users into Bad Ass Users.',
+        ],
+        'good_design_is_good_for_business_' => [
+            'title' => 'Good Design is Good for Business.',
+            'presenters' => ['lucien_frelin'],
+            'description' => ' Design is the not only the polish on a graphic. It is quickly becoming the factor that leads innovation.  Design in so many industries today is the defining advantage and concurrently a tool used to fend off competitors and create a owned differentiation.',
+        ],
+        'fine_tuning_sql_statements' =>
+        [
+            'title' => 'Fine-tuning SQL Statements',
+            'presenters' => ['randolf_julian'],
+            'description' => '',
+        ],
+        'legoman' => [
+            'title' => 'LegoMan',
+            'presenters' => ['brandon_beigay', 'michelle_brooks', 'chante_velez'],
+            'description' => ' Fun hands-on Lego building game (for on-site participants only) that will',
+        ],
+        'session2' => [
+            'title' => 'Refactoring',
+            'presenters' => ['chris_ryan'],
+            'description' => ' Refactoring code presentation with principals, rules, and real examples from our code bases. Examples will be mostly PHJP with some Javascript and NodeJS',
```

Comment:
yeah, i was leaving the descriptions as-is until they presenters send me their final ones.